<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComboBoxDialog</class>
 <widget class="QDialog" name="ComboBoxDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>341</width>
    <height>142</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QComboBox" name="comboBox"/>
   </item>
   <item row="0" column="1">
    <widget class="QFontComboBox" name="fontComboBox"/>
   </item>
   <item row="1" column="0">
    <widget class="QComboBox" name="comboBox_2"/>
   </item>
   <item row="1" column="1">
    <widget class="QFontComboBox" name="fontComboBox_2"/>
   </item>
   <item row="2" column="0">
    <widget class="QComboBox" name="comboBox_3"/>
   </item>
   <item row="2" column="1">
    <widget class="QFontComboBox" name="fontComboBox_3"/>
   </item>
   <item row="3" column="0">
    <widget class="QComboBox" name="comboBox_4"/>
   </item>
   <item row="3" column="1">
    <widget class="QFontComboBox" name="fontComboBox_4"/>
   </item>
   <item row="4" column="0">
    <widget class="QComboBox" name="comboBox_5"/>
   </item>
   <item row="4" column="1">
    <widget class="QFontComboBox" name="fontComboBox_5"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
